### PR TITLE
0.6.0 release preparation: ubuntu-21.04 cross-build and tests.

### PR DESCRIPTION
**Pull request by klihub**
Since we swapped out ubuntu-20.10 for 21.04, add cross-build and tests for the latter.

### CODE REVIEW

File: test/e2e/packages.test-suite/ubuntu-21.04/binsrc.var [hunk 1]
```diff
@@ -0,0 +1 @@
+packages/ubuntu-20.04
```
Comment (marquiz) [inline test/e2e/packages.test-suite/ubuntu-21.04/binsrc.var:1]: 20.04 or 21.04?

**Review by marquiz — COMMENTED** [review]
One question, otherwise looks good 👍

File: test/e2e/packages.test-suite/ubuntu-21.04/binsrc.var (hunk 1 above)
Comment (klihub) [reply test/e2e/packages.test-suite/ubuntu-21.04/binsrc.var]: Argh, yes should be 21.04. Fixed.

**Review by marquiz — APPROVED** [review]
lgtm

**Pull request merged**